src/main/java/decoder/Decoder.java

```diff
@@ -0,0 +1,25 @@
+package decoder;
+
+import com.google.protobuf.util.JsonFormat;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import proto.DataFormats.SerializedData;
+
+/** Decode data from the proto format to JSON. */
```

Comment:
is it mad bc of this extra star? I know im reaching, but cant think of why its upset